src/main/webapp/style.css

```diff
@@ -1,7 +1,57 @@
+body, html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  transition: box-shadow .25s;
+  border-radius: 2px;
+  margin-top: 30px;
+  width: 500px;
+}
+
+.card-container {
+  margin-right: 30px;
+  margin-left: 30px;
+  flex-shrink: 0;
+  flex-basis: 500px;
+}
+
+.card .card-description {
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 32px;
+}
+
+.card .card-title {
+  font-size: 20px;
+  font-weight: 300;
+  cursor: pointer;
+  line-height: 32px;
+  display: flex;
+}
+
+.card .card-content {
+  padding: 24px;
+  border-radius: 0 0 2px 2px;
+  width: 400px;
+}
+
+.card .card-content p {
```

Comment:
style: change this to `.card .card-content p, code ` if you change above to `<code>`